bee-hive/poetry.lock

```diff
@@ -0,0 +1,780 @@
+# This file is automatically @generated by Poetry 2.0.1 and should not be changed by hand.
```

Comment:
I recommend removing this file from the PR. This needs to be explicit since it already forms part of the PR. Including means running `poetry lock` often & impacting CI tests (including it helps with reproducible builds, but I don't feel that's needed at this point)

Reply:
this will fix the test failure - other comments can be handled in additional issues/prs as needed